en/tutorials-and-examples/blog/blog.rst

```diff
@@ -203,8 +202,680 @@ if the CakePHP welcome page looks a little funny (no images or CSS styles).
+            </td>
+            <td>
+                <?= $this->Form->postLink(
+                    'Delete',
+                    ['action' => 'delete', $article->id],
+                    ['confirm' => 'Are you sure?'])
+                ?>
+                <?= $this->Html->link('Edit', ['action' => 'edit', $article->id]) ?>
+            </td>
+        </tr>
+        <?php endforeach; ?>
+
+    </table>
+
+Using :php:meth:`~Cake\\View\\Helper\\FormHelper::postLink()` will create a link
+that uses JavaScript to do a POST request deleting our article.
+
+.. warning::
+
+    Allowing content to be deleted using GET requests is dangerous, as web
+    crawlers could accidentally delete all your content.
+
+.. note::
+
+    This view code also uses the ``FormHelper`` to prompt the user with a
+    JavaScript confirmation dialog before they attempt to delete an
+    article.
+
+Routes
+======
+
+For some, CakePHP's default routing works well enough. Developers
+who are sensitive to user-friendliness and general search engine
+compatibility will appreciate the way that CakePHP's URLs map to
+specific actions. So we'll just make a quick change to routes in
+this tutorial.
+
+For more information on advanced routing techniques, see
+:ref:`routes-configuration`.
+
+By default, CakePHP responds to a request for the root of your site
+(e.g., http://www.example.com) using its ``PagesController``, rendering
+a view called "home". Instead, we'll replace this with our
+ArticlesController by creating a routing rule.
+
+CakePHP's routing is found in **config/routes.php**. You'll want
+to comment out or remove the line that defines the default root
+route. It looks like this::
+
+    $routes->connect('/', ['controller' => 'Pages', 'action' => 'display', 'home']);
+
+This line connects the URL '/' with the default CakePHP home page.
+We want it to connect with our own controller, so replace that line
+with this one::
+
+    $routes->connect('/', ['controller' => 'Articles', 'action' => 'index']);
+
+This should connect users requesting '/' to the ``index()`` action of
+our ``ArticlesController``.
+
+.. note::
+
+    CakePHP also makes use of 'reverse routing'. If, with the above
+    route defined, you pass
+    ``['controller' => 'Articles', 'action' => 'index']`` to a
+    function expecting an array, the resulting URL used will be '/'.
+    It's therefore a good idea to always use arrays for URLs as this
+    means your routes define where a URL goes, and also ensures that
+    links point to the same place.
+
+Conclusion
+==========
+
+Now that you've created a basic CakePHP application, you can either continue to
+:doc:`/tutorials-and-examples/blog/part-two`, or follow the
+:doc:`/tutorials-and-examples/bookmarks/intro` which covers more features in the
+ORM and utitlizes :doc:`/bake/usage` to generates the code we wrote in this
+tutorial.  You can also peruse the :doc:`/topics` or `API
+<http://api.cakephp.org/3.2>`_ to learn more about CakePHP.
```

Comment:
the second mention if the api website seems to create an error in the build.
Also the 3.2 part can be removed so it will always be up to date.